common/governments/civics/hfe_civic_features.txt

```diff
@@ -2,70 +2,57 @@
 
 # starting civics
 hfe_civic_features_exodus = {
-	modification = no
-	potential = {
+#	modification = no
+	is_origin = yes
+	icon = "gfx/event_pictures/origins/TSP_exodus_origin_icon.dds"
+	picture = "GFX_HFE_TSP_exodus_origin"
+	preferred_planet_class_neighbor = no
+	max_once_global = yes
+	
+	potential = { always = yes }
+	
+	possible = {
 		authority = {
-			not = {
+			NOT = {
 				value = hfe_auth_brainworm
 			}
 		}
 	}
-	possible = {
-		civics = {
-			nor = {
-				text = hfe_civic_has_started
-				value = hfe_civic_features_early_colonisation
-				value = hfe_civic_features_early_contact
-				value = hfe_civic_features_fanatic_isolationist
-				value = hfe_civic_features_spiral_of_death
-				value = hfe_civic_features_sanctuary
 
-				# vanilla
-				# value = civic_mechanists
-				# value = civic_syncretic_evolution
-				#value = civic_post_apocalyptic
-				#value = civic_life_seeded
-			}
-		}
-	}
 	random_weight = {
 		base = -1
+		modifier = {
+			# civic has no effect after galaxy generation, disable for random generation
+			factor = 0
```

Comment:
-1*0 = 0
I made it -1 for a reason

Reply:
It's still negative one. It does not change it.

Reply:
Its not negative, its is zero

Reply:
Then what should I be dividing it by?

Reply:
Okay, done